modules/exploits/windows/persistence/linqpad_deserialization_persistence.rb

```diff
@@ -45,37 +48,43 @@ def initialize(info = {})
     register_options([
       OptString.new('LINQPAD_FILE', [true, 'Path to LINQPad executable on target\'s machine']),
       OptString.new('CACHE_PATH', [true, 'Path to cache file directory containing deserialized data']),
-      OptBool.new('CLEANUP', [false, 'Restore original cache file when exploit finish'])
     ])
   end
 
   # Simplify pulling the writable directory variable
 
   def check
-    if datastore['LINQPAD_PATH'].blank? || !file?(datastore['LINQPAD_PATH'])
-      return Exploit::CheckCode::Unknown('LINQPad binary not specified or doesn\'t exist')
+    if datastore['LINQPAD_FILE'].blank? || !file?(datastore['LINQPAD_FILE'])
```

Comment:
I'm unsure if the datastore option being blank means that the target software is `Safe` from us exploiting it. Would it make more sense to return `Unknown` here like before?